corehq/apps/app_manager/helpers/validators.py

```diff
@@ -419,6 +441,14 @@ def validate_with_raise(self):
                     'module': self.get_module_info(),
                 })
 
+        if hasattr(self.module, 'root_module_id') and self.module.root_module_id:
```

Comment:
Nitpick: `root_module_id` is from `ModuleBase` so it should always be available.